### PR TITLE
core/graph: add commands to increase/decrease movements speed

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

**Comment by radare** [comment]
wheelspeed is probably having a different meaning here.. but as long as it works with the graph canvas.. it would be good to handle it somehow too.

**Comment by ret2libc** [comment]
wheelspeed is used only when the movements are done with the mouse wheel.
do you want to have a way to increase/decrease that too?

**Comment by radare** [comment]
There’s already an eval var for this, no need to have keys imho. Also, maybe i prefer to use a single key
that toggles between two values of that delta. because pressing ‘>’ for a long time i can be a bit problematic
for the user

**Comment by ret2libc** [comment]
So would you prefer to use wheelspeed both for mouse wheel speed and for key speed?
Any suggestion for the key that toggles between the two values?

File: libr/core/graph.c [hunk 1]
```diff
@@ -14,6 +14,8 @@ static int mousemode = 0;
 #define MAX_NODE_WIDTH 18
 #define INIT_HISTORY_CAPACITY 16
 #define TITLE_LEN 128
+#define SLOW_MOV 1
+#define FAST_MOV 5
```
Comment (XVilka) [inline libr/core/graph.c]: what about putting that in the config variable? in the r_config?

Comment (ret2libc) [inline libr/core/graph.c]: sounds good to me

**Comment by radare** [comment]
we all miss wasd now :""

**Comment by ret2libc** [comment]
i've added the variables to set high/low speed values

**Comment by radare** [comment]
varnames look too long to me, what if low is set higher than high? i think that low should be always 1, so we only want to have the high one. isnt?

**Comment by radare** [comment]
Thanks for rebasing

**Pull request closed without merging**